Makefile

```diff
@@ -128,7 +128,13 @@ ib-init-package:
 	$(ZARF_BIN) package create -o build -a $(ARCH) --confirm . \
 		--set REGISTRY_IMAGE_DOMAIN="registry1.dso.mil/" \
 		--set REGISTRY_IMAGE="ironbank/opensource/docker/registry-v2" \
-		--set REGISTRY_IMAGE_TAG="2.8.2"
+		--set REGISTRY_IMAGE_TAG="2.8.2" \
+		--set AGENT_IMAGE_DOMAIN="registry1.dso.mil/" \
+		--set AGENT_IMAGE="ironbank/opensource/defenseunicorns/zarf/zarf-agent" \
+		--set AGENT_IMAGE_TAG="v0.29.2" \
```

Comment:
We could test this separately later, but right now we will still want to use the local tag since it may break things specific to Big Bang and may get out of sync if we ever have breaking agent changes.